tests/e2e/specs/in-game/game-over/rematch.spec.js

```diff
@@ -337,7 +337,7 @@ describe('Creating And Updating Casual Games With Rematch', () => {
       p1Hand: [ Card.TEN_OF_SPADES, Card.ACE_OF_DIAMONDS ],
       p1Points: [],
       p1FaceCards: [],
-    });
+    }, false, true);
```

Comment:
I wonder if it would make sense to combine these into a single object argument so each value would be a key/value pair. That might make it easier to know what these values are.

If we go that route, we should probably combine the pNum value into the same object and do the same for both assertGameState and loadGameFixture. Thoughts?